src/sections/post-event/post-event.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+
+import style from "./post-event.module.scss"
+
+import Icon from "../../components/icon"
+import Spacer from "../../components/spacer"
+import TextBlock from "../../components/text-block"
+import logo from "../../assets/images/logo.svg"
+import Typography from "../../components/typography"
+
+const PostEvent = () => (
+  <div className={style.root}>
+    <Spacer size="xs" />
+    <TextBlock headline="!🎉 Thank you all for #jscc22 🎉!" />
```

Comment:
I would prefer JSCraftCamp instead of jscc22, mainly for marketing reasons...
```suggestion
    <TextBlock headline="!🎉 Thank you all for JSCraftcamp 2022 🎉!" />
```